custom/icds_reports/management/commands/sql_scripts/nos_of_deaths.sql

```diff
@@ -0,0 +1,17 @@
+
+SELECT
+  convert_to(state, 'UTF8') AS state,
+  convert_to(district, 'UTF8') AS district,
+  convert_to(awc, 'UTF8') AS AWC,
+  death_month,
+  count(*) as no_of_deaths
+FROM (
+      SELECT awc_location.state_name AS state,
+             awc_location.district_name AS district,
+             awc_location.awc_name AS awc,
+             date_trunc('month',person_case.date_death) as death_month
+      FROM "%(person_table_name)s" AS person_case
```

Comment:
since the two table names are constant there is no need to parameterize them

Reply:
I think it is needed as the table names `config_report_icds-cas_static-awc_location_88b3f9c3` which i guess after calculating hash with domain and table name. so It feels safe to just calculate and pass it.

Reply:
yeah, definitely makes sense. I was thinking these were more clear like the dashboard names but makes sense.